lib/handlers.js

```diff
@@ -12,18 +12,17 @@ function executeHandler (
 function getHandler (
   module /* : string */,
   method /* : string */
-) /* : Function | void */ {
-  let handler
+) /* : Promise<Function | void> */ {
```

Comment:
Any reason why `getHandler()` went from being synchronous to being a `Promise`r? The statements are still synchronous.